frontend/apps/app/app/global-error.tsx

```diff
@@ -6,7 +6,9 @@ import { useEffect } from 'react'
 
 export default function GlobalError({
   error,
-}: { error: Error & { digest?: string } }) {
+}: {
+  error: Error & { digest?: string }
+}) {
```

Comment:
I wanted to reduce the code differences as much as possible, but I could not stop the formatter from changing its behavior, so I'm committing